### ISSUE TITLE
setDefaultEmailValues should maybe not set signature and greeting, 

### ISSUE THREAD

**Issue by thomasf** [matcornic/hermes#25]
Becuse sometimes they should not be set at all.

**Comment by matcornic** [comment]
Good point, it's because of zero values. I think I can come up with a solution that will set default values and let user override it at the same time.
